<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1186</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QGroupBox" name="team1">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>40</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>TEAM1</string>
      </property>
      <widget class="QLabel" name="t1nameLB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TEAM1 NAME</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="t1nameLE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="t1settingBtn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>setting</string>
       </property>
      </widget>
      <widget class="QLabel" name="t1imageLB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>181</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>390</y>
        <width>105</width>
        <height>15</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Play List</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="list">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>410</y>
        <width>421</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="team2">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>40</y>
        <width>201</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>TEAM2</string>
      </property>
      <widget class="QLabel" name="t2nameLB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>TEAM2 NAME</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="t2nameLE">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="t2settingBtn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>setting</string>
       </property>
      </widget>
      <widget class="QLabel" name="t2imageLB">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>181</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_pause">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>550</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>❚❚</string>
      </property>
     </widget>
     <widget class="QPushButton" name="detectBtn">
      <property name="geometry">
       <rect>
        <x>1040</x>
        <y>540</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Detecting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_prev">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>550</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>◀◀</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_del">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>570</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Del</string>
      </property>
     </widget>
     <widget class="QSlider" name="bar">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="croppersonBtn">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>540</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Crop Person</string>
      </property>
     </widget>
     <widget class="QLabel" name="playtime">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>510</y>
        <width>264</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Running time</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>540</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_forward">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>550</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>▶▶</string>
      </property>
     </widget>
     <widget class="QLabel" name="state">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>510</y>
        <width>264</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Media Status</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>0</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Videosave</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="downloadbar">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>0</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_play">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>550</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>▶</string>
      </property>
     </widget>
     <widget class="QSlider" name="vol">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>490</y>
        <width>84</width>
        <height>15</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="downurl">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>721</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QVideoWidget" name="view" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>721</width>
        <height>441</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_stop">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>550</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>■</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1186</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menumenu">
    <property name="title">
     <string>menu</string>
    </property>
    <addaction name="actionmain"/>
    <addaction name="actioncamera"/>
   </widget>
   <addaction name="menumenu"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionmain">
   <property name="text">
    <string>main</string>
   </property>
  </action>
  <action name="actioncamera">
   <property name="text">
    <string>camera</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
